extension/landing-zone/README.md

```diff
@@ -0,0 +1,23 @@
+# VSI add-on for landing-zone
+
+A module creating a VSI in an existing landing zone VPC.
+
+## Infrastructure
+
+This module creates and configures the following infrastructure:
```

Comment:
Same as https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/pull/529/files#r1325592303 around the use of the word "module"